frontend/admin/src/js/components/classification/ClassificationPage.tsx

```diff
@@ -11,17 +12,26 @@ export const ClassificationPage: React.FC = () => {
   const intl = useIntl();
 
   return (
-    <DashboardContentContainer>
-      <PageHeader icon={TagIcon}>
-        {intl.formatMessage({
+    <>
+      <SEO
+        title={intl.formatMessage({
           defaultMessage: "Classifications",
-          id: "hnyrdl",
-          description:
-            "Heading displayed above the Classification Table component.",
+          id: "xJm72U",
+          description: "Page title for the classification index page",
         })}
-      </PageHeader>
-      <ClassificationTableApi />
-    </DashboardContentContainer>
+      />
+      <DashboardContentContainer>
+        <PageHeader icon={TagIcon}>
+          {intl.formatMessage({
+            defaultMessage: "Classifications",
+            id: "hnyrdl",
+            description:
+              "Heading displayed above the Classification Table component.",
+          })}
```

Comment:
Can the duplicate strings be turned into `pageTitle` variable too?

Reply:
Goo catch! You should see there are all resolved in [reduce duplicate intl strings](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5024/commits/a60a6630586bfb6ab864c59411cb5602500ef970)